agent/engine/dockerclient/dockerclientfactory.go

```diff
@@ -14,13 +14,19 @@
 package dockerclient
 
 import (
-	"errors"
-
 	"github.com/aws/amazon-ecs-agent/agent/engine/dockeriface"
+	"github.com/aws/amazon-ecs-agent/agent/utils"
 	log "github.com/cihub/seelog"
 	docker "github.com/fsouza/go-dockerclient"
+	"github.com/pkg/errors"
 )
 
+const (
+	// minAPIVersionKey is the docker.Env key for min API version
+	minAPIVersionKey = "MinAPIVersion"
```

Comment:
Can you also document here that this is supported in API versions `>=1.25` as per https://docs.docker.com/engine/api/version-history/#v125-api-changes ?